chore(request-validator): add more explanation for allowed_content_types of request-validator (#4683)
Kong/docs.konghq.com: app/_hub/kong-inc/request-validator/_index.md

      value_in_examples: null
      datatype: Set of string elements
      description: |

>         List of allowed content types. The value can be configured with the `charset` parameter(e.g. `application/json; charset=UTF-8`).

Reviewer: ```suggestion
        List of allowed content types. The value can be configured with the `charset` parameter. For example, `application/json; charset=UTF-8`.
```

        <br>**Note:** 
        Body validation is only done for `application/json` and skipped for any other allowed content types. 

>         Only supports one parameter at most. If a request is sending more than one with the Content-Type header, only the first parameter will be evaluated and the rest will be truncated. (Please note that `application/json` is not match with `application/json; charset=UTF-8`)

Reviewer: ```suggestion
        Only one parameter is supported at the most. If a request is sending more than one parameter with the Content-Type header, only the first parameter is evaluated and the rest are truncated. Note that `application/json` doesn't match with `application/json; charset=UTF-8`.
```

>         The type, subtype, parameter names, and the value of the charset parameter are not case sensitive based on the RFC explaination.

Reviewer: ```suggestion
        The type, subtype, parameter names, and the value of the charset parameter are not case sensitive based on the RFC explanation.
```
